x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/no_models.tsx

```diff
@@ -37,21 +41,28 @@ export const NoModelsPanel: React.FC = () => {
                 'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.noModels.description',
                 {
                   defaultMessage:
-                    'You have no trained models available. Please follow the documenation to add trained ml models to your cluster.',
+                    'You have no supported trained models available. Please follow the documentation to add trained machine learning models to your cluster.',
+                }
+              )}
+            </p>
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.noModels.supportedTasksDescription',
+                {
+                  defaultMessage:
+                    'Inference pipelines support the following types of pytorch models: {supportedTasksList}',
```

Comment:
I kind of like the minimalist approach in the previous approach, without this list 🙈.

Reply:
I'm not opposed to removing/not adding this paragraph. Was just trying to provide as much context as possible.

Reply:
If the context is helpful absolutely include.